client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dashboard-v2.tsx

```diff
@@ -97,6 +101,27 @@ export default function SitesDashboardV2() {
 		[ setSitesViewState ]
 	);
 
+	useEffect( () => {
+		let filterOptions: AgencyDashboardFilterOption[] = [];
+		const filterMap: AgencyDashboardFilterOption[] = [
+			'all_issues',
+			'backup_failed',
+			'backup_warning',
+			'threats_found',
+			'site_disconnected',
+			'site_down',
+			'plugin_updates',
+		];
+		for ( let i = 0; i < filterMap.length; i++ ) {
+			if ( sitesViewState.filters.some( ( filter ) => filter.value === i + 1 ) ) {
+				filterOptions = [ filterMap[ i ] ];
+			}
+		}
+
+		updateDashboardURLQueryArgs( { search: sitesViewState.search } );
+		updateDashboardURLQueryArgs( { filter: filterOptions } );
```

Comment:
```suggestion
		updateDashboardURLQueryArgs( { search: sitesViewState.search,  filter: filterOptions } );
```

Reply:
and I think that `updateDashboardURLQueryArgs( { filter: filterOptions } );` should work too.

Reply:
Confirmed that works, thanks!